lib/install-php-ubuntu.sh

```diff
@@ -5,17 +5,22 @@ set -eo pipefail
 release=$(lsb_release -cs)
 version=$1
 
+# see https://www.cyberciti.biz/faq/how-to-speed-up-apt-get-apt-command-ubuntu-linux/
+sudo apt-get install software-properties-common
+sudo add-apt-repository ppa:apt-fast/stable
+
 sudo apt-get update
+sudo apt-get -y install apt-fast
```

Comment:
it seems even apt-fast might already be installed:

![grafik](https://user-images.githubusercontent.com/120441/65435581-08891780-de21-11e9-93e6-9f1e1916782e.png)

Reply:
Thank you so much!